bower.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "leaflet-color-markers",
+  "homepage": "https://github.com/pointhi/leaflet-color-markers",
+  "authors": [
+    "Thomas Pointhuber <thomas.pointhuber@gmx.at>"
+  ],
+  "description": "color variations of the standard leaflet marker",
+  "main": "js/leaflet-color-markers.js",
+  "license": "BSD-2-Clause",
+  "dependencies": {
+    "leaflet": ">=0.7.0"
```

Comment:
technically, this repo has no dependencies

Reply:
The Javascript file in the repo uses Leaflet code (specifically, a dependency on L.Icon), which is why I put this here... but of course I understand the icons can potentially be useful in other contexts as well. I wouldn't object to dropping this section, or changing "dependencies" to "devDependencies" to show that the relationship is optional (though that's not exactly the right semantic use of devDependencies).